src/fu-util-common.c

```diff
@@ -1236,6 +1236,30 @@ fu_util_device_flag_to_string(guint64 device_flag)
 	return NULL;
 }
 
+static gchar *
+fu_util_request_flag_to_string(guint64 request_flag)
```

Comment:
Does the client really need different strings for generic vs non generic messages or generic vs non generic images?  I feel like this should just be two things - message and image.

Reply:
Maybe not the GUID client -- but the json output does. Maybe `Message (generic)` and `Message (custom)` is better?

Reply:
Yes that makes better sense.